node/chainSimulator/chainSimulator.go

```diff
@@ -93,7 +93,9 @@ func (s *simulator) createChainHandlers(args ArgsChainSimulator) error {
 			shardIDStr = "metachain"
 		}
 
-		node, errCreate := s.createTestNode(outputConfigs.Configs, shardIDStr, outputConfigs.GasScheduleFilename, args.ApiInterface, args.BypassTxSignatureCheck, args.InitialRound)
+		node, errCreate := s.createTestNode(
+			outputConfigs.Configs, shardIDStr, outputConfigs.GasScheduleFilename, args.ApiInterface, args.BypassTxSignatureCheck, args.InitialRound, args.MinNodesPerShard, args.MetaChainMinNodes,
```

Comment:
put all parameters on a new line? Also, we might consider using a sort of argument struct DTO